deployment/nifi/nifi.yml

```diff
@@ -137,7 +151,7 @@ spec:
         readinessProbe:
           tcpSocket:
             port: cluster
-          initialDelaySeconds: 60
+          initialDelaySeconds: 90
```

Comment:
After increasing the `readinessProbe.initialDelaySeconds`, it will be worked. And I configure `200` seconds.

Reply:
The NiFi K8s cluster is completed now after doing above step:

![image](https://github.com/user-attachments/assets/61bd2c4e-5daa-4403-b2c8-50bfac713ba7)